ARM-wvd-templates/HCI/HybridCompute/Readme.md

```diff
@@ -0,0 +1,27 @@
+# Azure Virtual Desktop with Stack HCI
```

Comment:
note: there is also a readme right above this one

idea:  should we organize this parent tree in a way that makes sense why we have 2 sets of templates?   We now have a HybridCompute set of templates (target=GA) and we now have a set of AzureStackHCI templates (target=publicPreview).  kushmeen?  thoughts?

Reply:
will make changes for GA